hubblestack/extmods/modules/nebula_osquery.py

```diff
@@ -317,6 +327,120 @@ def get_top_data(topfile):
 
     return ret
 
+def mask_passwords_inplace(object_to_be_masked):
+    '''
+    It masks the passwords present in 'object_to_be_masked'. Uses "mask.yaml" as
+    a reference to find out the list of blacklisted strings or objects.
+    Note that this method alters "object_to_be_masked".
+    '''
+    try:
+
+        mask = {}
+        topfile = 'salt://hubblestack_nebula_v2/top.mask'
```

Comment:
instead of writing file paths in function definitions, these should be written as constants in some file. By looking at the path, I feel there will, someday, be a hubblestack_nebula_v3, due to which you will have to make changes in code. This increases maintainability of the code.